support/python/src/nebula_sdk/webhook.py

```diff
@@ -23,6 +24,16 @@ class PortParseError(Exception):
 
 
 class WebhookServer:
+    """Instantiate a WebhookServer class to handle incoming webhook payloads
+
+    Trigger webhook handlers should expect POST requests to the top level ('/')
```

Comment:
To be fair, this actually depends on the implementation of the webhook - I'd probably leave this out.